zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImpl.java

```diff
@@ -0,0 +1,355 @@
+     * @param email the email of the user to check.
+     * @return null
+     * @throws UnsupportedOperationExceptions will error if invoked.
+     */
+    @Override
+    public boolean canEdit(String email) throws IOException {
+        throw new UnsupportedOperationExceptions("JWT Permissions service error for canEdit no longer required");
+    }
+
+    /**
+     * @param user the {@link User} to check.
+     * @return null
+     * @throws UnsupportedOperationExceptions will error if invoked.
+     */
+    @Override
+    public boolean canEdit(User user) throws UnsupportedOperationExceptions {
+        throw new UnsupportedOperationExceptions("JWT Permissions service error for canEdit no longer required");
+    }
+
+    /**
+     * implemented as part of session migration to JWT
+     * will not be required once jwt has been migrated but will error if invoked
+     *
+     * @param email   the email of the user to permit the permission to.
+     * @param session the {@link Session} of the {@link User} granting the permission.
+     * @throws IOException If a filesystem error occurs.
+     */
+    @Override
+    public void addEditor(String email, Session session) throws IOException {
+        throw new UnsupportedOperationExceptions("JWT Permissions service error for addEditor no longer required");
+    }
+
+    /**
+     * implemented as part of session migration to JWT
+     * will not be required once jwt has been migrated but will error if invoked
+     *
+     * @param email   the email of the user to revoke the permission from.
+     * @param session the {@link Session} of the {@link User} revoking the permission.
+     * @throws UnsupportedOperationExceptions will error if invoked.
+     */
+    @Override
+    public void removeEditor(String email, Session session) throws UnsupportedOperationExceptions {
+        throw new UnsupportedOperationExceptions("JWT Permissions service error for removeEditor no longer required");
+    }
+
+    /**
+     * Determines whether the subscribed session has viewing authorisation to collection
+     *
+     * @param session               the {@link Session} to get the user details from.
+     * @param collectionDescription the {@link CollectionDescription} of the {@link Collection} to check.
+     * @return null
+     * @throws IOException If a filesystem error occurs.
+     */
+    @Override
+    public boolean canView(Session session, CollectionDescription collectionDescription) throws IOException {
+        List<Integer> teams = convertGroupsToTeams(session);
+        try {
+            AccessMapping accessMapping = permissionsStore.getAccessMapping();
+            Set<Integer> collectionTeams = accessMapping.getCollections().get(collectionDescription.getId());
+            if (collectionTeams == null || collectionTeams.isEmpty()) {
+                return false;
+            }
+            return teams.stream().anyMatch(t -> collectionTeams.contains(t));
+        } catch (IOException ignored) {
```

Comment:
If something goes wrong here its because there was an issue reading the accessing mapping file which IMO is something we want/need to be aware of.

Can you update this throw that exception please.